app/bundles/PluginBundle/Config/config.php

```diff
@@ -139,7 +139,7 @@
                 'alias' => 'integration_campaign_status',
             ],
         ],
-        'other' => [
+        'helpers' => [
```

Comment:
`'helpers'` are auto tagged with `templating.helper` for use in templates. Was that intended here? https://github.com/mautic/mautic/blob/5f04ac46a42564faa86310aff1aa1764de37a981/app/bundles/CoreBundle/DependencyInjection/MauticCoreExtension.php#L54

Reply:
Other than this question, the new command works great.